### DISCUSSION TITLE
Should the Common Media Library be expanded to languages other than Typescript?

### DISCUSSION THREAD

**Discussion by littlespex** [streaming-video-technology-alliance/common-media-library#63] [Polls]
The Common Media Library main use is in web based media players. The utilities provided could also be useful on other platforms but would require the code to be ported to other languages: Java for Android, Python and/or Go for servers, Swift for iOS.

**Comment by Beraliv** [comment]
@littlespex as an option, you may write a universal code in Rust, building it for an HTML5 environment (using WASM/asm.js) and iOS/Android native apps (using foreign function interface, or FFI, to call native C API).
